src/js/overview.js

```diff
@@ -8,7 +8,10 @@ $(function () {
 	var nbrPercent = d3.format("%") // e.g 53%
 	var nbrSI = d3.format("s") // e.g 5.1k
 
+	var divNode = d3.select("body").node();
 
+	/* colorScheme is defined in util/color.js */
+	var color = window.util.colorScheme()
 
 	/* adjust font size on a per-label granularity */
 	var labelScale = (label) => {
```

Comment:
Ideally we should replace this by some function that tries to fit a label inside a rectangle:

```js
function labelScale(label, rect) {
    var defaultCharWidth = 14 /* px */
    var length = label.length
    var size = label.length * defaultCharWidth
    return defaultCharWidth * (rectangle.width / size) * 0.9
}
```

But I don't know if we can figure out how much space there is inside each facet.